src/views/ObserveGames/ObserveGames.tsx

```diff
@@ -120,7 +120,11 @@ export class ObserveGames extends React.PureComponent<ObserveGamesProperties, an
         this.setPage(this.state.page + 1);
     }}}
     setPage = (ev_or_page) => {{{
-        let page = parseInt(typeof(ev_or_page) === "number" ? ev_or_page : (ev_or_page.target as any).value) || 1;
+        if ((ev_or_page.target as any).value === "") {
```

Comment:
Duh! I didn't even test those buttons afterwards. Sorry for the caveman coding. ^.^